### PR TITLE
Development process

### PR DESCRIPTION
**Pull request by deian40**

### CODE REVIEW

**Review by npecko — APPROVED** [review]

**Comment by deian40** [comment]
[Write down the development process in Github](https://trello.com/c/ysvf9sJm/27-write-down-the-development-process-in-github)

**Comment by deian40** [comment]
[Write down the development process in Github](https://trello.com/c/ysvf9sJm/27-write-down-the-development-process-in-github)

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+		* git status - shows status and current branch
+		* git checkout -b <branchname> - creates new branch from current branch
+	* [UI](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository)
+
+
+## 2. Commit and push chnages to new feature branch
+* bash
+	* git status - shows status and chnages made to files
+	* git add <filename> - add file for commit
+	* git commit -m "message" - commits the added files and
```
Comment (svetoslaw) [inline README.md:17]:
```suggestion
	* git commit -m "message" - commits the added files and 
	* git push --set-upstream origin <your_branch_name> - creates and sets an upstream branch for the feature
```

**Review by svetoslaw — APPROVED** [review]

**Comment by deian40** [comment]
[Write down the development process in Github](https://trello.com/c/ysvf9sJm/27-write-down-the-development-process-in-github)

**Pull request merged**